client/src/pages/AdvCalendar.js

```diff
@@ -0,0 +1,165 @@
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components/macro';
+
+import Modal from '../components/Modal';
+import Data from '../data.json';
+import '../lib/basic.css';
```

Comment:
If you use CSS files with or instead of styled-components you should create a `css` folder where you store all your CSS.